### PR TITLE
Convert ADropdown to use AFloatingMenu

### PR DESCRIPTION
**Pull request by rwharrisjr**
Additionally adds ability to set `component` of either `ATag` or `AButton`
Extends `ATag` to handle a `dropwdown` variant

### CODE REVIEW

File: framework/components/ATag/ATag.js
Comment (androche) [inline framework/components/ATag/ATag.js:1]: We kept this file in for props.  Otherwise they are empty
Screenshot 2024-10-15 at 9 09 07 AM

Comment (rwharrisjr) [reply framework/components/ATag/ATag.js:1]: Ah, I thought we might have fixed that

**Comment by androche** [comment]
Cool although with this approach can the tags still be different colors and variations?  I'm wondering if it makes more sense to fix up the tag itself?
Screenshot 2024-10-15 at 2 23 45 PM

**Comment by rwharrisjr** [comment]
@androche maybe if we added in a `componentProps` prop? Either that or we can split it out into ATagDropdown, but that seems excessive

**Pull request merged**